### ISSUE TITLE
[Question] What's the usage of `reschedule`?

### ISSUE THREAD

**Issue by L-M-Sherlock** [open-spaced-repetition/ts-fsrs#125]
https://github.com/open-spaced-repetition/ts-fsrs/blob/ba9fa37cfa51e31e222eed1249df8ae78b35f7c8/src/fsrs/fsrs.ts#L414-L445

 The input should be a series of reviews of the same card, isn't it?

**Comment by ishiko732** [comment]
Only reschedule based on the current state, aiming to recalculate the interval when switching `enable_fuzz`. It was approved in #72 that this design, but it has become somewhat redundant after implementing long-term/short-term scheduling. Should it be redesigned?

**Comment by L-M-Sherlock** [comment]
Yes. I think it should be something described in https://github.com/open-spaced-repetition/ts-fsrs/issues/112#issue-2461399492

> To solve this, the reviews would need to be replayed on top of each other to compute intermediary/eventual state.

**Comment by ishiko732** [comment]
> To solve this, the reviews would need to be replayed on top of each other to compute intermediary/eventual state.

No problem, but the implementation will need to be delayed, as I am currently busy with other matters.
